.projectile

```diff
@@ -0,0 +1 @@
+-/vendor
```

Comment:
Ha yeah I'd say remove it from the PR and achieve what you're trying to do using one of the options I mentioned above.

But this is a good idea, I'm going to ignore vendor directories too :smile: